src/query/frame_change.hpp

```diff
@@ -73,8 +88,8 @@ class FrameChangeCollector {
 
   // Add tracking key to cache later value
   CachedValue &AddTrackingKey(const std::string &key) {
-    tracked_values_.emplace(key, CachedValue{});
-    return tracked_values_[key];
+    tracked_values_.emplace(key, tracked_values_.get_allocator().GetMemoryResource());
```

Comment:
emplace returns an iterator and bool, no need to do at() after, just use the iterator.
p.s. the bool is false if the key already exists, but the iterator always points to the correct element.